cpp/models/abm/testing_strategy.h

```diff
@@ -178,14 +135,13 @@ class TestingScheme
      * @brief Runs the TestingScheme and potentially tests a Person.
      * @param[inout] rng Person::RandomNumberGenerator for the Person being tested.
      * @param[in] person Person to check.
-     * @param[in] location Location to check.
      * @param[in] t TimePoint when to run the scheme.
      * @return If the person is allowed to enter the Location by the scheme.
      */
-    bool run_scheme(Person::RandomNumberGenerator& rng, Person& person, const Location& location, TimePoint t) const;
+    bool run_scheme(Person::RandomNumberGenerator& rng, Person& person, TimePoint t) const;
 
 private:
-    std::vector<TestingCriteria> m_testing_criteria; ///< Vector with all TestingCriteria of the scheme.
```

Comment:
This one was suggested by Daniel because we don't have any TestingScheme with multiple TestingCriteria in the simulation (only in tests). In this way, we avoid using vectors in exchange for more schemes to be defined.